index.js

```diff
@@ -17,14 +17,19 @@ export class AbortError extends Error {
 	}
 }
 
-const createRetryContext = (error, attemptNumber, options) => {
+const createRetryContext = ({error, attemptNumber, startTime, options}) => {
```

Comment:
As I needed to add an arg, I took the opportunity to make this an object, as it seemed easier to work with.